src/data/dtos/users/customer-phones.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { Type } from 'class-transformer';
+import { IsArray, IsNotEmpty, ValidateNested } from 'class-validator';
+export class CustomerPhoneDto {
+  @IsNotEmpty()
+  readonly number: string;
+
+  @IsNotEmpty()
+  readonly contact_person: string;
+}
+
+export class CreateFrameworksDto {
+  @IsNotEmpty()
+  readonly number: string;
+
+  @IsNotEmpty()
```

Comment:
add `@IsString()` for string checker for all dto

Reply:
Done